cfgov/v1/jinja2tags/__init__.py

```diff
@@ -98,6 +98,34 @@ def render_stream_child(context, stream_child):
     return Markup(unescaped)
 
 
+def unique_id_in_context(context):
+    """For a given request, return a unique ID each time this method is
```

Comment:
Per [the docstrings PEP](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings), would you mind changing this so that it consists

> of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.